docs/advanced-topics/transport-configuration.md

```diff
@@ -0,0 +1,15 @@
+# Transport configuration
+Currently, Azure SignalR Service supports configuring the transport type between clients and service. You can specify the transport type for each client by setting `ServiceOptions.TransportTypeDetector`.
+The `TransportTypeDetector` is a function which takes the `HttpContext` during negotiation as parameter and returns a bit mask combining one or more transport type values.
+
+For example, if you want to disable the Server-Sent Event transport for all clients, you can set a transport type detector when you add Azure SignalR Service as a dependency:
```

Comment:
disable SSE -> enable WebSockets and LongPolling only. Use positive description to match the code?